script/script.js

```diff
@@ -0,0 +1,32 @@
+//.go-to-gallery height
+$(document).ready( adaptiveHeight );
+$(window).resize( adaptiveHeight );
+function adaptiveHeight(){
+  var a = $('.go-to-gallery');
```

Comment:
I don't understand what does var `a` mean. You need to change name of variable. It's syntactically correct, its purpose is not evident.